src/store/plugins/external-layers.plugin.js

```diff
@@ -50,7 +50,7 @@ async function updateExternalLayer(store, externalLayer, projection) {
         }
 
         updatedExternalLayer.isLoading = false
-        store.commit('updateLayer', { layer: updatedExternalLayer })
+        store.dispatch('updateLayer', updatedExternalLayer)
```

Comment:
So that it may go through the layer validation present, post-commit, in the action.
But that isn't mandatory, just that I prefer not touching mutation much, and using action whenever I can (only store modules themselves should use mutations)
It will greatly ease the future migration to pinia not to use mutation in the code outside of store modules.

Reply:
Ok good to know I'll try to stick to actions then